lib/kubeclient/common.rb

```diff
@@ -477,5 +482,16 @@ def http_options(uri)
 
       options.merge(@socket_options)
     end
+
+    def expired_token_message(http_code, message)
```

Comment:
Consider changing the name of the method to something like `check_token_expiration` as it adds to the mesasge.